startup/28-optimization_detector_functions.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+
+
+def start_detector(detector):
+    """Set-up and start detector"""
+    pass
+
+
+def read_detector(detector, pos):
+    """
+    Read detector
+    Uses motor positions to create a gaussian type signal. Gaussian center is 27.2,
+    width is .2, and amplitude is 10
+    Parameters
+    ----------
+    detector : detector object
```

Comment:
Latest sphinx versions will not process the line correctly if there is a blank space before `:`.
```suggestion
    detector: detector object
```